services/pulsar/pulsar-downloads.service.js

```diff
@@ -0,0 +1,47 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+import { metric } from '../text-formatters.js'
+import { nonNegativeInteger } from '../validators.js'
+import { pulsarPurple } from './pulsar-helper.js'
+
+const schema = Joi.object({
+  downloads: nonNegativeInteger,
+})
+
+export default class PulsarDownloads extends BaseJsonService {
+  static category = 'downloads'
+
+  static route = { base: 'pulsar/downloads', pattern: ':packageName' }
```

Comment:
We use `dt` (shorthand for "download total")  in many similar badges, I'm wondering whether we should align for consistency?